app/views/shared/_citations.html.erb

```diff
@@ -15,12 +15,29 @@
 <% end %>
 
 <% content_for(:gscholar_meta) do %>
+  <meta name="description" content="<%= @presenter.description.first.truncate(200) rescue @presenter.title.first %>" />
   <meta name="citation_title" content="<%= @presenter.title.first %>" />
+
   <% @presenter.creator.each do |creator| %>
   <meta name="citation_author" content="<%= creator %>" />
   <% end %>
+
+  <% unless @presenter.date_created.blank? %>
   <meta name="citation_publication_date" content="<%= @presenter.date_created.first %>" />
+  <% end %>
+
+  <% unless @presenter.download_url.blank? %>
   <meta name="citation_pdf_url" content="<%= @presenter.download_url %>" />
+  <% end %>
+
+  <% unless @presenter.keyword.blank? %>
+  <meta name="citation_keywords" content="<%= @presenter.keyword.join('; ') %>" />
+  <% end %>
+
+  <% unless @presenter.publisher.blank? %>
+  <meta name="citation_publisher" content="<%= @presenter.publisher.join %>" />
```

Comment:
I'm not aware of the syntax, but do you want a delimiter here?

Reply:
Good question.  I don't really know the syntax either and there aren't easily found good docs on these tags that I could find.  Probably safest to follow the same as for keywords (`'; '`) even though this might be an edge case?

Reply:
I agree that it's an edge case and think `; ` probably makes the most sense